src/views/settings/view_outfit_editor.cpp

```diff
@@ -4,6 +4,12 @@
 
 namespace big
 {
+	static int range(int lower_bound, int upper_bound)
+	{
+		std::srand(std::chrono::system_clock::now().time_since_epoch().count());
```

Comment:
Setting the seed every time you call it?

![image](https://user-images.githubusercontent.com/24669514/226321061-3cba5a9a-e70d-4d3f-bf66-090728917e25.png)

Reply:
Ah, yeah, I moved it into main